python/tvm/_ffi/_cython/object.pxi

```diff
@@ -106,6 +106,10 @@ cdef class ObjectBase:
         def __set__(self, value):
             self._set_handle(value)
 
+        def __del__(self):
+            if self.chandle != NULL:
```

Comment:
cross check impl of `__dealloc__ ` and see if there will be issue of double free